lib/ui/window.dart

```diff
@@ -729,7 +729,7 @@ class Window {
     final Zone registrationZone = Zone.current;
 
     return (Uint8List data) {
-      registrationZone.runUnaryGuarded(callback, data.buffer.asByteData());
+      registrationZone.runUnaryGuarded(callback, data != null ? data.buffer.asByteData() : null);
```

Comment:
data?.buffer.asByteData() instead of data != null ? ...

Reply:
Done.